main/src/ca/uwaterloo/flix/api/lsp/SemanticTokenType.scala

```diff
@@ -36,11 +36,14 @@ sealed trait SemanticTokenType {
     case SemanticTokenType.Operator => 5
     case SemanticTokenType.Parameter => 6
     case SemanticTokenType.Property => 7
-    case SemanticTokenType.Method => 8
-    case SemanticTokenType.Namespace => 9
-    case SemanticTokenType.Type => 10
-    case SemanticTokenType.TypeParameter => 11
-    case SemanticTokenType.Variable => 12
+    case SemanticTokenType.Regexp => 8
+    case SemanticTokenType.Method => 9
+    case SemanticTokenType.Namespace => 10
+    case SemanticTokenType.Number => 11
+    case SemanticTokenType.String => 12
+    case SemanticTokenType.Type => 13
+    case SemanticTokenType.TypeParameter => 14
+    case SemanticTokenType.Variable => 15
```

Comment:
I don't think you can change this without changing server\src\handlers.ts so that poses an issue. Hmm